package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "codecov",
-  "version": "0.0.5",
+  "version": "0.0.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "codecov",
-      "version": "0.0.5",
+      "version": "0.0.0",
```

Comment:
was this intended?

Reply:
Looks like this was just never updated after this change: https://github.com/codecov/vscode/pull/42#discussion_r1557909150

TLDR: Yes it's intended